subprojects/docs/src/docs/userguide/pluginReference.adoc

```diff
@@ -0,0 +1,109 @@
+
+<<java_library_plugin,Java Library>>::
+Provides support for building a Java library.
+
+<<groovy_plugin,Groovy>>::
+Provides support for building any type of http://groovy-lang.org/[Groovy] project.
+
+<<scala_plugin,Scala>>::
+Provides support for building any type of https://www.scala-lang.org/[Scala] project.
+
+<<play_plugin,Play>>::
+Proivdes support for building, testing and running https://www.playframework.com/[Play] applications. 
+
+<<antlr_plugin,ANTLR>>::
+Provides support for generating parsers using http://www.antlr.org/[ANTLR].
+
+=== Packaging and distribution
+
+<<application_plugin,Application>>::
+Provides support for building JVM-based, runnable applications.
+
+<<war_plugin,WAR>>::
+Provides support for building and packaging WAR-based Java web applications.
+
+<<ear_plugin,EAR>>::
+Provides support for building and packaging Java EE applications.
+
+<<osgi_plugin,OSGi>>::
+Provides support for creating https://www.osgi.org/[OSGi] packages.
+
+<<maven_publish,Maven Publish>>::
+Provides support for <<publishing_overview,publishing artifacts>> to Maven-compatible repositories.
+
+<<ivy_publish,Ivy Publish>>::
+Provides support for <<publishing_overview,publishing artifacts>> to Ivy-compatible repositories.
+
+<<maven_plugin,Legacy Maven Plugin>>::
+Provides support for publishing artifacts using the <<legacy_publishing,legacy mechanism>> to Maven-compatible repositories.
+
+<<distribution_plugin,Distribution>>::
+Makes it easy to create ZIP and tarball distributions of your project.
+
+<<java_library_distribution_plugin,Java Library Distribution>>::
+Provides support for creating a ZIP distribution of a Java library project that includes its runtime dependencies.
+
+=== Code analysis
+
+<<checkstyle_plugin,Checkstyle>>::
+Performs quality checks on your project’s Java source files using http://checkstyle.sourceforge.net/index.html[Checkstyle] and generates associated reports.
+
+<<findbugs_plugin,FindBugs>>::
+Performs quality checks on your project’s Java source files using http://findbugs.sourceforge.net/[FindBugs] and generates associated reports.
+
+<<pmd_plugin,PMD>>::
+Performs quality checks on your project’s Java source files using http://pmd.sourceforge.net/[PMD] and generates associated reports.
+
+<<jdepend_plugin,JDepend>>::
+Performs quality checks on your project’s Java source files using http://clarkware.com/software/JDepend.html[JDepend] and generates associated reports.
+
+<<jacoco_plugin,JaCoCo>>::
+Provides code coverage metrics for your Java project using http://www.eclemma.org/jacoco/[JaCoCo].
+
+<<codenarc_plugin,CodeNarc>>::
+Performs quality checks on your Groovy source files using http://codenarc.sourceforge.net/index.html[CodeNarc] and generates associated reports.
+
+=== IDE integration
+
+<<eclipse_plugin,Eclipse>>::
+Generates Eclipse project files for the build that can be opened by the IDE. It can also be used to fine tune Buildship's import process for Gradle builds.
```

Comment:
A link to Buildship perhaps?

Reply:
Ah, I thought I forgot to update a link. I also need to clarify that there are multiple plugins.

 - [ ] Link to Buildship project page
 - [ ] Rephrase to avoid implying that there is only one Eclipse plugin (there are two at the moment)